<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>779</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>779</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hydro Visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="statusBar">
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="openFileBtn">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Open...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fdirEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>directory path</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string># steps:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Save All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="filePrevBtn">
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileNextBtn">
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="fnameEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>filename</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Step size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="displayCheckbox">
          <property name="text">
           <string>Show Display Window</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="queryCheckbox">
          <property name="text">
           <string>Show Query Window</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="view1dThetaBtn">
          <property name="text">
           <string>1D (fixed theta)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view1dRBtn">
          <property name="text">
           <string>1D (fixed r)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view1dBtn">
          <property name="text">
           <string>No 1D</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_display">
          <property name="text">
           <string>Display:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Set Background Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bgColorBtn">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="gridLcheckbox">
          <property name="text">
           <string>Show Grid</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="scaleCheckbox">
          <property name="text">
           <string>Show Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="logScaleCheckbox">
          <property name="text">
           <string>Log Scale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="rotateBtn">
          <property name="text">
           <string>Rotate 90 deg</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mirrorCheckbox">
          <property name="text">
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mirror4xCheckbox">
          <property name="text">
           <string>4x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Enforce 1:1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axesLayout">
        <item>
         <widget class="QLabel" name="axesLabel">
          <property name="text">
           <string>Axes (x0,x1,y0,y1): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="axis_x0_edit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="axis_x1_edit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="axis_y0_edit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="axis_y1_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Colormap:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="colorbarCheckbox">
          <property name="text">
           <string>Show Colorbar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cbarCycleBtn">
          <property name="text">
           <string>Cycle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorbarResetBtn">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Gamma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Center:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="value">
           <number>49</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Slope:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cbarmin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cbarmax">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cboundReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Set Variable to Display:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="variableLayoutTop">
        <item>
         <widget class="QLabel" name="varTopLbl">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="variableLayoutBottom">
        <item>
         <widget class="QLabel" name="varBotLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="statusBar_bottom">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Close All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionOptions"/>
    <addaction name="actionSave_Config"/>
    <addaction name="actionLoad_Config"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
